content/docs/started/getting-started-gke.md

```diff
@@ -72,8 +72,15 @@ Create an OAuth client ID to be used to identify Cloud IAP when requesting acces
     export CLIENT_ID=<CLIENT_ID from OAuth page>
     export CLIENT_SECRET=<CLIENT_SECRET from OAuth page>
     ```
+## Deploy Kubeflow on Kubernetes Engine Using The Ui
 
-## Deploy Kubeflow on Kubernetes Engine
+1. Open [https://deploy.kubeflow.cloud/](https://deploy.kubeflow.cloud/#/deploy) in your web browser
+
+1. Fill out the form 
+
+1. Click Create Deployment
+
+## Deploy Kubeflow on Kubernetes Engine Using The Command Line
```

Comment:
Fix capitalisation - Use sentence case instead of title case:

Deploy Kubeflow on Kubernetes Engine using the command line